clients/v1/clients.proto

```diff
@@ -26,9 +26,12 @@ service UsersService {
   rpc GetUserRoles(GetUserRolesRequest) returns (GetUserRolesResponse);
 }
 
+// Exactly one attribute can be set.
```

Comment:
Does it have to? i.e. I want to get a user with ID `xxx-yyy-zzz` AND email `aa@example.com`, if I don't get a result, I know such user doesn't exist? 😂 but maybe that's an unreal use case.

Reply:
Yeah, I think it's better to just bug out, especially because the data layer doesn't have a query that accepts arbitrary parameters

Reply:
It also means we won't have to implement a query that is not useful at the data layer